packages/modules/viewer-react/src/components/app-ui/providers/BackstageItemsProvider.ts

```diff
@@ -36,8 +35,8 @@ export class BackstageItemsProvider implements UiItemsProvider {
             IModelApp.localization.getLocalizedString(
               backstageItem.labeli18nKey
             ),
-            backstageItem.subtitle,
-            backstageItem.icon
+            backstageItem.subtitle?.toString(),
```

Comment:
It's annoying that createStageLauncher doesn't accept ConditionalString as a type for subtutle and icon. It would prevent this nastiness.